example/index.ts

```diff
@@ -0,0 +1,57 @@
+import { rAF, forceV, applyForce, vector as Vector, Entity } from "../src";
+
+interface UseGravityArgs {
+  moverMass: number;
+  attractorMass: number;
+  attractor: Vector<number>;
+}
+
+const applyForceForStep = (
+  config: UseGravityArgs,
+  mover: Entity,
+  entity: Entity
+) => {
+  const force = forceV({
+    ...config,
+    mover: mover.position
+  });
+
+  return applyForce({
+    force,
+    moverMass: config.moverMass,
+    acceleration: mover.acceleration,
+    velocity: mover.velocity,
+    position: mover.position
+  });
+};
+
+const run = (config: UseGravityArgs) => {
+  const { start, stop } = rAF();
+
+  start((timestamp, lastFrame, mover) => {
+    const diffTime = timestamp > lastFrame + 64 ? 0 : lastFrame;
+    const steps = Math.floor(timestamp - diffTime);
+
+    let entity: Entity;
+
+    if (steps > 0) {
+      for (let i = 0; i < steps; i++) {
+        entity = applyForceForStep(config, mover, entity);
+      }
+    } else {
+      entity = applyForceForStep(config, mover, entity);
+    }
+
+    return entity;
```

Comment:
So this is our time step piece. @bmathews and I were talking about this yesterday. `react-spring` does this interesting thing where, for each frame, it determines the ms difference between the previous frame and the current one. Then, for each millisecond, it applies the physics calculations. For example, if 16ms elapsed b/t the previous and last frame, it would run the spring acceleration calculations 16 times. B/c we're working with two dimensional vectors I think we may be getting bogged down here – `react-spring` just uses one dimension: https://github.com/react-spring/react-spring/blob/master/src/animated/FrameLoop.ts#L67-L83

@littlemilkstudio what is your advice for this piece of our animation equation? Basically, as I understand it, we want to "continually apply the force" to match real world physics as closely as possible. I could see us either using a single dimension for `useGravity` and then using two dimensions for `useGravity2d` or reducing the steps by half (so, for example, we apply the force every 2ms rather than every 1ms).

Reply:
This was basically gonna be my approach for making the lerp frame rate independent. Cool to see this is the solution that react-spring came to.

Reply:
I'm not sure if I'm following your question 100%. If we could chat at some point so I could get a better understanding of your proposed approaches, that would be awesome.